tools/tracer/README

```diff
@@ -0,0 +1,60 @@
+THIS DOCUMENT CONTAINS INFORMATION ON PRODUCTS IN THE DESIGN PHASE OF DEVELOPMENT.
+
+Copyright(c) 2014-2020 Intel Corporation. All Rights Reserved.
```

Comment:
rename the file to README.md for it to render in a nice way

Reply:
done